lambdas/jwtverification/src/test/java/uk/gov/di/ipv/cri/passport/jwtverification/JwtVerificationHandlerTest.java

```diff
@@ -105,4 +139,41 @@ void shouldReturn400IfFailedToParseJWT() throws JsonProcessingException {
         assertEquals(ErrorResponse.FAILED_TO_PARSE_SHARED_ATTRIBUTES_JWT.getCode(), error.get("code"));
         assertEquals(ErrorResponse.FAILED_TO_PARSE_SHARED_ATTRIBUTES_JWT.getMessage(), error.get("message"));
     }
+
+    @Test
+    void shouldReturn400IFClientIdIsNotSet() throws JsonProcessingException {
```

Comment:
Slight casing issue with the test name for 'If" word
```suggestion
    void shouldReturn400IfClientIdIsNotSet() throws JsonProcessingException {
```

Reply:
Refactored